trulens_eval/trulens_eval/feedback/groundtruth.py

```diff
@@ -3,13 +3,17 @@
 
 import numpy as np
 import pydantic
+import scipy.stats as stats
+from sklearn.metrics import ndcg_score
+from sklearn.metrics import roc_auc_score
 
 from trulens_eval.feedback.provider import Provider
-from trulens_eval.utils.generated import re_0_10_rating
+from trulens_eval.utils.generated import re_configured_rating
+from trulens_eval.utils.imports import OptionalImports
 from trulens_eval.utils.imports import REQUIREMENT_BERT_SCORE
 from trulens_eval.utils.imports import REQUIREMENT_EVALUATE
 from trulens_eval.utils.imports import REQUIREMENT_OPENAI
-from trulens_eval.utils.imports import OptionalImports
```

Comment:
Importing `OptionalImports` is still required

Reply:
it's there (see 3 lines above)- I think it's just a isort thing